apps/dashboard/app/models/project.rb

```diff
@@ -253,6 +253,15 @@ def make_dir
     false
   end
 
+  def updated_permission
+    # Read and execute to others
+    project_dataroot.chmod(0775)
```

Comment:
Seems like we should use 0750 for a little bit of safety.

Reply:
Updated